wildfly-openssl.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: wildfly-openssl
+  version: 2.2.5
+  epoch: 0
+  description: OpenSSL-based Java Secure Socket Extension implementation
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - build-base
+      - openssl-dev
+      - openjdk-11-default-jvm
+      - openjdk-11
+      - maven
+  environment:
+    JAVA_HOME: /usr/lib/jvm/java-11-openjdk
+
+pipeline:
+  - working-directory: /home/build/nativelib
+    pipeline:
+      - uses: git-checkout
+        with:
+          repository: https://github.com/wildfly-security/wildfly-openssl-natives
+          tag: 2.2.2.Final
+          expected-commit: d7c3dfa07684ff08ff704685cbf2bf35834b2c04
+      # Install native binary JAR to $HOME/.m2 so it can be used later
+      - runs: |
+          mvn install
+
+  - working-directory: /home/build/final
+    pipeline:
+      - uses: git-checkout
+        with:
+          repository: https://github.com/wildfly-security/wildfly-openssl
+          tag: 2.2.5.Final
+          expected-commit: 9e58cff606974ec33918b7874951364fe02c58e5
+      - runs: |
+          mvn package
+
+  # Do something with the WildFly JAR, located at /home/build/final/combined/target/wildfly-openssl-${{package.version}}.Final.jar
+  - runs: |
+      ls -alh /home/build/final/combined/target/wildfly-openssl-${{package.version}}.Final.jar
+
+update:
+  enabled: true
```

Comment:
Since there are two unrelated source artifacts that do not use the same version, we need to manually handle updates here.

Reply:
Thanks a lot @kaniini just disabled it